avpipe.go

```diff
@@ -1446,6 +1454,19 @@ func Probe(params *XcParams) (*ProbeInfo, error) {
 		probeInfo.StreamInfo[i].FieldOrder = AVFieldOrderNames[AVFieldOrder(probeArray[i].field_order)]
 		probeInfo.StreamInfo[i].Profile = int(probeArray[i].profile)
 		probeInfo.StreamInfo[i].Level = int(probeArray[i].level)
+		probeInfo.StreamInfo[i].SideData.DisplayMatrix.Rotation = float64(probeArray[i].side_data.display_matrix.rotation)
+
+		dict := (*C.AVDictionary)(probeArray[i].tags)
+		dictCount := (int)(C.dict_count(dict))
+		if dictCount > 0 {
+			probeInfo.StreamInfo[i].Tags = make(map[string]string, dictCount)
+			for c := 0; c < dictCount; c++ {
```

Comment:
I doubt it'll ever matter, but this could possibly get slow if we have a large number of dictionary entries because this is O(n^2). If it becomes an issue we can write a more efficient iterator to grab this stuff.

Reply:
This was removed - I made it work directly using the av_dict_get() iterator form Go